opentelemetry-exporters-newrelic/build.gradle.kts

```diff
@@ -3,6 +3,7 @@ dependencies {
     val openTelemetryVersion = "0.6.0"
 
     api("com.newrelic.telemetry:telemetry:$newRelicTelemetrySdkVersion")
+    api("org.slf4j:slf4j-api:1.7.26")
```

Comment:
I *think* this can be `implementation`, since we don't expose the slf4j apis as a part of our api.